### ISSUE TITLE
Feature request: VimRuntime command

### ISSUE THREAD

**Issue by CGamesPlay** [junegunn/fzf.vim#1056]
I've added a few simple, short commands to my vimrc to enable me to quickly navigate my own vim runtime files, and one in particular may be of interest to include in the general ff-vim community: `VimRuntime` lists all loaded vim scripts and allows you to open them.

```vim
" FZF for all vim runtime files
command! VimRuntime call fzf#run(fzf#wrap({
  \ 'source': split(substitute(execute('scriptnames'), ' *\d*: ', '', 'g'), "\n"),
  \ 'options': ['--prompt', 'Vim> ', '--nth=2'],
  \ }))
```

I think this command is worth adding to the main ff-vim distribution because it is similar in scope to the already-included `Buffers`, `Commands`, `Colors`, etc.  commands.

If this command is actually to be included, ideally the `substitute` call would be removed, showing the user the script number alongside each result, but doing this requires writing a custom sink plugin which is not possible from outside of the plugin (it's basically a copy of `s:buffer_line_handler`, which makes use of many other `s:` functions).

If there is interest, I'm happy to submit a PR implementing.

**Comment by rafaelnp** [comment]
@CGamesPlay Good idea. However, when I typed a filename, the file list gets empty.
Using Neovim v0.4.4 on Linux.

**Comment by CGamesPlay** [comment]
Not totally sure, but I do notice that my local version of this command has `--nth=1`, so this command might work:

```vim
" FZF for all vim runtime files
command! VimRuntime call fzf#run(fzf#wrap({
  \ 'source': split(substitute(execute('scriptnames'), ' *\d*: ', '', 'g'), "\n"),
  \ 'options': ['--prompt', 'Vim> ', '--nth=1'],
  \ }))
```
